docs/UsersGuide/source/ContributorsGuide.rst

```diff
@@ -0,0 +1,369 @@
+
+    * Bug fixes (Bug fixes on what part of the code?)
+    * Changes to surface scheme (What kind of changes? What surface scheme?)
+
+**DESCRIPTION OF CHANGES:** The first line of the description should be a single-line "purpose" for this change. Note the type of change (i.e., bug fix, feature, enhancement, textonly). Summarize the problem, proposed solution, and required changes. If this is an enhancement or new code, describe why the change is important.
+
+**DOCUMENTATION:** Developers adding a new capability should include documentation on how to use this capability. This documentation can be included in the appropriate section of the PR message or submitted as a file attachment. Developers should consider updating the existing ``.rst`` documentation files in the ``ufs-srweather-app`` repository. If this is not possible, they can submit an issue to indicate that documentation updates are required based on the documentation in their PR submission. 
+
+Tips, Best Practices, and Protocols to Follow When Issuing a PR
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Indicate urgency.** If a PR is particularly urgent, this information should be provided in the PR Description section, and multiple code management team members should be tagged to draw attention to this issue.
+* **Indicate the scope of the PR.** If the PR is extremely minor (e.g., change to the README file), indicate this in the PR message. If it is an extensive PR, test it on as many platforms as possible, and stress the necessity that it be tested on systems for which you do not have access.
+* **Clarify in the PR message where the code has been tested.** At a minimum, code should be tested on the platform where code modification has taken place. It should also be tested on machines where code modifications will impact results. If the developer does not have access to these platforms, this should be noted in the PR. 
+* **Follow separation of concerns.** For example, module loads are only handled in the appropriate modulefiles, Rocoto always sets the work directory, j-jobs make the work directory, and ex-scripts require the work directory to exist.
+..
+    COMMENT: Originally, it mentioned style guidelines for the regional_workflow, but where are those located?
+* **Label PR status appropriately.** If the PR is not completely ready to be merged please add a “draft” or “do not merge” label to the PR. 
+* **Target subject matter experts among the code management team.** When possible, tag team members who are familiar with the modifications made in the PR so that the committee can provide effective and streamlined PR reviews and approvals.
+* **Schedule a live code review** if the PR is exceptionally complex in order to brief members of the code management team on the PR either in-person or through a teleconference.
+..
+    COMMENT: Are these last two points possible for contributors outside of NOAA?? Is there a process for this??
+
```

Comment:
Yes, but can anyone schedule a live code review? How would they do that if they're outside of NOAA? Would we be wondering who this random person is and ignore their request? Or is this actually likely to happen?

Reply:
That's a good question.  We've yet to run into this issue, but we've also yet to have a live code review request either.  Anyone could request one, but it would likely be someone with whom we've worked, or who has contacted us before.  We don't have a process outlined on how a live code review would work, but simply requesting one in the PR description would be a good first step.  Then we could work out a time, etc.

Reply:
It could also be raised the standing SRW code management review meetings on Thrs.